src/renderer/shared/api/price/coingecko/CoingeckoAdapter.ts

```diff
@@ -0,0 +1,48 @@
+import { AssetId, Currency, PriceObject, PriceAdapter, PriceItem, PriceRange } from '../common/types';
+import { getCurrencyChangeTitle } from '../common/utils';
+import { COINGECKO_URL } from './consts';
+
+export class CoinGeckoAdapter implements PriceAdapter {
```

Comment:
`PriceAdapter ` is a `Type`
It's more correct to make it an `interface`

By the way, why you decided to use a class? 
Right now (and I hope it will be the same in the future) it doesn't store any state
and provides only 2 pure functions
You didn't want to go `hook style` like we did in services? Or there is another reason? 🤔